test/e2e/serial/workload_placement_test.go

```diff
@@ -108,23 +109,75 @@ var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
 			//TODO : filter nodes with SingleNumaNode policy?
 
 			By("Calculate the requirements")
-			requiredRes, err := getWorkloadRequirements(nodeResourceTopologies, suitableNodes)
+			requiredRes, err = getWorkloadRequirements(nodeResourceTopologies, suitableNodes)
 			if err != nil {
 				Skip(fmt.Sprintf("not suitable request found: %v", err))
 			}
 
-			By("Prepare the pod object with the requirements")
-			pod = objects.NewTestPodPause(testFixture.Namespace.Name, "testpod")
-			pod.Spec.SchedulerName = schedulerName
-			pod.Spec.Containers[0].Resources.Limits = requiredRes
-
 		})
 
-		FIt("Dummy test right now", func() {
+		When("creating a guaranteed pod with one container", func() {
+			var guaranteedPod *corev1.Pod
+			var updatedPod *corev1.Pod
+
+			BeforeEach(func() {
+				By("Prepare the pod object with the requirements")
+				guaranteedPod = objects.NewTestPodPause(testFixture.Namespace.Name, "guaranteedtestpod")
+				guaranteedPod.Spec.SchedulerName = schedulerName
+				guaranteedPod.Spec.Containers[0].Resources.Limits = requiredRes
+				guaranteedPod.Spec.Containers[0].Resources.Requests = requiredRes
+
+				By("Create pod")
+				err := testFixture.Client.Create(context.TODO(), guaranteedPod)
+				Expect(err).NotTo(HaveOccurred())
+
+				By("Wait until pod is up&running")
+				const podRunningTimeout = 5 * time.Second
+				const podRunningPollPeriod = 500 * time.Millisecond
+				Eventually(func() error {
+					//Get pod (waiting to be scheduled)
+					updatedPod = &corev1.Pod{}
+					podKey := client.ObjectKeyFromObject(guaranteedPod)
+					err = testFixture.Client.Get(context.TODO(), podKey, updatedPod)
+					if err != nil {
+						return err
+					}
+
+					if updatedPod.Status.Phase != corev1.PodRunning {
+						return fmt.Errorf("Pod `%s/%s` still not running", updatedPod.GetNamespace(), updatedPod.GetName())
+					}
+
+					return nil
+				}, podRunningTimeout, podRunningPollPeriod).ShouldNot(BeNil(), "Pod dit not become Running")
+			})
+
+			AfterEach(func() {
+				By("Delete pod")
+				err := testFixture.Client.Delete(context.TODO(), guaranteedPod)
```

Comment:
if you create the pods needed by the test in the namespace managed by the fixture (and we should always do that), cluster will clean up everything automatically when we shut down the fixture, because shutting down the fixture deletes the namespace.

Reply:
Yes, I know ... I was thinking about using the same context for the burstable and best-effort pods ( moving the limits and request assignment part to the test case body)

Reply:
uhm, ok, but still the explicit delete seems unnecessary. Or is it?

Reply:
well as we will be using the same namespace, and environment to all the tests we could avoid collisions changing the pod names or deleting them after use. 
Of course we change the test suite structure so we could first check if the cluster has the pre-requisites for our test suite and then create and destroy namespaces for each test.

Reply:
ok, let's see the final layout and we will iterate over there.